spec.html

```diff
@@ -46541,7 +46542,7 @@ <h1>Valid Chosen Reads</h1>
           1. Let _readLen_ be the number of elements of _readValue_.
           1. If _chosenLen_ &ne; _readLen_, then
             1. Return *false*.
-          1. If _chosenValue_[_i_] &ne; _readValue_[_i_] for any integer value _i_ in the range 0 through _chosenLen_, exclusive, then
+          1. If _chosenValue_[_i_] &ne; _readValue_[_i_] for some integer _i_ in the interval from 0 (inclusive) to _chosenLen_ (exclusive), then
```

Comment:
why some instead of any?

Reply:
"any" is ambiguous; it can be read to mean "every". (As in "for any integer n, n can be written as a unique product of primes" - that's a claim about _every_ integer.) So [pedantic grammarians](https://faculty.math.illinois.edu/~west/grammar.html) [see point 19] suggest using "some" in contexts like this.

Reply:
Yeah natural language quantification is wild.